funfact/initializers.py

```diff
@@ -125,3 +125,22 @@ def __call__(self, shape):
         shape = as_tuple(shape)
         std = (self.scale / shape[self.axis])**0.5
         return std * self.distribution(shape)
+
+
+def vectorize_initializer(initializer, post: bool = True):
```

Comment:
Also, 'vectorize_initializer' might be a bit too wordy to type. I understand that this is to avoid conflict with the `vectorize` method for the tensor expressions. How about we steal `vmap` from JAX? Or maybe call it `stack` since we are literally stacking up lower dimensionality outputs?

Reply:
What do you think about renaming it to 'append' but keeping it as boolean? Since we only ever have two options I think a boolean makes most sense, or do you think there will ever be a reason to set an intermediate axis?

Reply:
Yeah this sounds like a good choice!